adodbapi/test/setuptestframework.py

```diff
@@ -85,12 +85,9 @@ def makemdb(testfolder, mdb_name):
             from win32com.client.gencache import EnsureDispatch
 
             win32 = True
-        except ImportError:  # perhaps we are running IronPython

```

Comment:
The try-except block can be eliminated. It's only reason for existence was to automatically detect IronPython. The import of EnsureDispatch can be simple.